tooling/nargo/src/lib.rs

```diff
@@ -49,6 +49,7 @@ pub fn prepare_package(package: &Package, file_reader: Box<FileReader>) -> (Cont
     let mut context = Context::new(fm, graph);
 
     let crate_id = prepare_crate(&mut context, &package.entry_path);
+    context.root_crate_id = crate_id.clone();
```

Comment:
```suggestion
    context.root_crate_id = crate_id;
```

This is `Copy` so a clone is unnecessary.